tests/freeipamock.py

```diff
@@ -148,12 +147,12 @@ def deskprofileconfig_mod(self, ipadeskprofilepriority):
 
     def user_show(self, user):
         if user in self.data.users:
-            return {u"result": {}, u"value": six.text_type(user), u"summary": None}
+            return {u"result": {}, u"value": str(user), u"summary": None}
```

Comment:
btw the unicode legacy literal (u'value') can be dropped
https://www.python.org/dev/peps/pep-0414/